doc/getting_started/setup_fpga.md

```diff
@@ -47,35 +45,24 @@ This can be spliced into the bitstream to overwrite the testing boot ROM as desc
 However, if you do not want to do the splicing yourself, both versions of the bitstream are available in the download as `*.bit.orig` and `*.bit.splice` (containing the test ROM and the mask ROM respectively).
 The metadata for the latest bitstream (the approximate creation time and the associated commit hash) is also available as a text file and can be [downloaded separately](https://storage.googleapis.com/opentitan-bitstreams/master/latest/latest.txt).
 
-### Create an FPGA bitstream
-
-Synthesizing a design for an FPGA board is done with the following commands.
+### Build an FPGA bitstream
 
-The FPGA build will pull in a program to act as the boot ROM.
-This must be built before running the FPGA build.
-This is pulled in from the `sw/device/lib/testing/test_rom` directory (see the `parameters:` section of the `hw/top_earlgrey/chip_earlgrey_cw310.core` file).
+Synthesizing a design for an FPGA board is simple with Bazel.
+While Bazel is the entry point for kicking off the FPGA sythesis, under the hood, it invokes FuseSoC, the hardware package manager / build system supported by OpenTitan.
+During the build process, the boot ROM is baked into the bitstream.
+As mentioned above, we maintain two boot ROM programs, one for testing (_test ROM_), and one for production (_mask ROM_).
 
-To build it:
+To build an FPGA bitstream with the _test ROM_, use:
 ```console
 cd $REPO_TOP
-./meson_init.sh
-ninja -C build-out all
+bazel build //hw/bitstream/vivado:fpga_cw310_test_rom
 ```
 
-Next, the actual FPGA implementation can be started.
-In the following example we synthesize the Earl Grey design for the ChipWhisperer CW310 board using Xilinx Vivado {{< tool_version "vivado" >}}.
-
+To build an FPGA bitstream with the _mask ROM_, use:
 ```console
-. /tools/Xilinx/Vivado/{{< tool_version "vivado" >}}/settings64.sh
 cd $REPO_TOP
-./meson_init.sh
-ninja -C build-out all
-fusesoc --verbose --cores-root . run --flag=fileset_top --target=synth lowrisc:systems:chip_earlgrey_cw310
+bazel build //hw/bitstream/vivado:fpga_cw310_mask_rom
 ```
-The `fileset_top` flag used above is specific to the OpenTitan project to select the correct fileset.
-
-The resulting bitstream is located at `build/lowrisc_systems_chip_earlgrey_cw310_0.1/synth-vivado/lowrisc_systems_chip_earlgrey_cw310_0.1.bit`.
```

Comment:
We seem to have lost documentation of where to find the artifacts. Someone building bitstreams is not necessarily going to be running in a totally bazel-directed way, so we should replace this line with an equivalent.